docs/faq/lukso/general-information.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_label: 'General Information'
+sidebar_position: 1
+---
+
+# General Information
+
+### What is the LUKSO Network?
+
+LUKSO is a Layer-1 blockchain network dedicated to existing and coming digital lifestyles and creative use cases. It is built on the EVM framework and compatible with all other platforms or protocols built on Ethereum.
+
+### Will LUKSO be the main platform to integrate LSPs?
+
+The LUKSO blockchain will be the first mover for LUKSO Standard Proposals. In conjunction with the surrounding ecosystem, Universal Profiles greatly enhance the interaction and convenience of managing LSPs. This advantage stems from the **seamless integration of Universal Profiles**. With a dedicated network like LUKSO, we can implement well-thought-out, upgradable standards directly from the start. This clean-cut approach allows us to bypass the challenges of retrofitting standards into an existing system.
+
+### Who defined the LUKSO token supply?
```

Comment:
maybe you can add a link to this article which also explains this: https://medium.com/lukso/its-happening-the-genesis-validators-are-coming-ce5e07935df6